horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -68,6 +68,13 @@
         % particular projection-projection pair of transformations,
         % optimized for specific projection-projection pair of classes
         do_generic;
+        % testing property. Normaly transformation from source to target
+        % coordinate system in cut can be optimized as each transformation
+        % is described by a transformation matrix and the final
+        % transformation is the production of these matrices.
+        % if the property set to true, the transformation performed in two
```

Comment:
two or N? in principle can have multiple matrices combining

Reply:
clarified